README.md

```diff
@@ -37,11 +37,19 @@ those versions. On the other hand, this can become cumbersome if we get into the
 - We think it's highly likely a consumer app will also use that library
 - We know that if a consumer app uses a different version of the dependency, it
   will cause issues.
+- There are corresponding Pods that need to be installed for iOS
 
 If you are a LifeOmic customer, building a mobile app, and see a dependency that
 you feel should be a peer dependency, reach out to us at
 development@lifeomic.com.
 
+### Export everything, except default
+
+Try not to use `export default`. If we use named exports everywhere, we can
+`export * from ...` in index.ts files that facilitate what is available in the
+npm lib. Try to export any/everything that might be buseful for reuse, and try
```

Comment:
```suggestion
npm lib. Try to export any/everything that might be useful for reuse, and try
```

Reply:
👍 I'll fix this in an upcoming "cleanup" PR - trying to get these merged to stop the conflict-fest for Bruce's graphql PR.